.github/workflows/windows-build-and-test.yaml

```diff
@@ -111,6 +111,12 @@ jobs:
         ~/PostgreSQL/${{ matrix.pg }}/bin/pg_isready -U postgres -d postgres --timeout=30
         ~/PostgreSQL/${{ matrix.pg }}/bin/psql -U postgres -d postgres -c 'CREATE USER root SUPERUSER LOGIN;'
         ~/PostgreSQL/${{ matrix.pg }}/bin/psql -U postgres -d postgres -c 'SHOW log_filename;SHOW data_directory;SELECT version();'
```

Comment:
This line is already showing all 3 values. Should remove those that you want to print explicitly below.

Reply:
For some reason it only printed version last time, so I added separate lines. I'm trying to upload the postgres log from windows CI checks, but don't know its location. The `dist_param` test is timing out for some reason on windows, in the simplest query `insert into metrics_dist select * from metrics_local`...